Converters/Revit/Speckle.Converters.RevitShared/ToHost/TopLevel/ModelCurveToSpeckleTopLevelConverter.cs

```diff
@@ -33,7 +33,12 @@ public override DB.ModelCurve[] Convert(SOBR.Curve.ModelCurve target) =>
       // Curves must be bound in order to be valid model curves
       if (!curve.IsBound)
       {
-        curve.MakeBound(speckleLine.domain.start ?? 0, speckleLine.domain.end ?? Math.PI * 2);
+        if (speckleLine.domain.end - speckleLine.domain.start <= 0)
```

Comment:
This doesn't seem to be exactly the same condition as we were checking. Any reason we need to check for 0 length domain?

Reply:
Claire and I rewrote this, the goal here, as I understand it, is to call `MakeBound` when its not a bounded interval (i.e. is zero length)
I'll give this a test, but I think its good.